vamb/__main__.py

```diff
@@ -1221,21 +1229,14 @@ def cluster_and_write_files(
                 file=file,
                 sep="\t",
             )
+            plural = 's' if len(cluster.members) > 1 else ''
+            logger.info(f"\tCluster {i} created with {len(cluster.members)} contig{plural}.\n")
```

Comment:
This is too verbose; Vamb may generate tens of thousands of clusters, which will completely dominate the file. I recommend just deleting this, since users can watch the clusters.tsv file for progress. However, instead, you could compute the number of contigs, and then log a line whenever say, 10% of the contigs have been clustered.